### DISCUSSION TITLE
Checking in without Raspberry Pi

### DISCUSSION THREAD

**Discussion by tobsowo** [alfio-event/alf.io#1216] [Q&A]
Is it possible to check-in using the mobile app scanner and then print from a PC with a thermal printer?

We're organising a free open source event and budget is very limited for us to get a raspberry pi, the screen and the scanner and also the thermal printer.

Also, is tye recent version compatible with Raspberry Pi 4?

**Comment by cbellone** [comment]
Hi!

> Is it possible to check-in using the mobile app scanner and then print from a PC with a thermal printer?

no. We'd like to extend Alf.io-PI in the future to run also on Mac/Linux PC (optionally Windows, but I am not sure if we can manage it) but as for now, the only installation target is RaspberryPI

We might also release an utility to generate a multi-page PDF with all badges, useful if you want to pre-print them.

> Also, is tye recent version compatible with Raspberry Pi 4?

not yet. The RaspberryPI 4-compatible OS includes a version of [CUPS](https://en.wikipedia.org/wiki/CUPS) which does not support the DYMO labels that we use.
We are evaluating alternatives to CUPS, but it will take time. Any help with testing/development would be highly appreciated.

**Reply by tobsowo** [reply]
Thank you.
Well noted.
